src/common/pf_lldp.h

```diff
@@ -363,6 +363,22 @@ void pf_lldp_send (pnet_t * net);
  */
 void pf_lldp_tx_restart (pnet_t * net, bool send);
 
+/**
+ * @internal
+ * Parse LLDP frame buffer
+ * 
+ * Exposed for unit-testing purposes.
+ *
+ * @param buf              In:    LLDP data.
+ * @param len              In:    Length of LLDP data.
+ * @param lldp_peer_info   Out:   Record holding parsed LLDP information.
+ * @return 0 if parsing is successful, -1 on error.
+ */
+int pf_lldp_parse_packet (
```

Comment:
In other parts where we expose functions for testing, we have kept the docstring in the c-file. 
The header file looks for example like:
```
/************ Internal functions, made available for unit testing ************/

bool pf_cmina_is_stationname_valid (const char * station_name, uint16_t len);

bool pf_cmina_is_netmask_valid (pnal_ipaddr_t netmask);
```

Reply:
Done.